### PR TITLE
feat(develop): adds zkEVM wallet tutorial

### PR DESCRIPTION
**Pull request by cerberushades**

### CODE REVIEW

File: docs/zkEVM/wallet/usage-guide.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  - wallet
+  - bridge
+image: https://wiki.polygon.technology/img/thumbnail/polygon-zkevm.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Developers can deploy their existing contracts to the zkEVM, and users can deposit assets from Ethereum and transact off-chain. For moving assets across chains, you will need to use the zkEVM Bridge. The bridge interface is available on [the Wallet Suite page](https://wallet.polygon.technology/). 
+
+In this tutorial, you will learn how to bridge assets from Ethereum Mainnet to Polygon zkEVM and vice-versa.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:19]: Better if this was the first statement on the page

File: docs/zkEVM/wallet/usage-guide.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+You can either watch the **video tutorial** here or follow the **step-by-step guide** provided below.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:21]:
```suggestion
You can either follow the **step-by-step guide** or watch the **video tutorial** provided below.
```

File: docs/zkEVM/wallet/usage-guide.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<div align="center">
+  <video loop autoplay width="70%" height="70%" controls="true" >
+    <source type="video/mp4" src="/img/zkevm/zkevmwallettestnet.mp4"></source>
+    <p>Your browser does not support the video element.</p>
+  </video>
+</div>
+
+## Step-by-Step Guide
+
+On Polygon Wallet Suite website, you can find a tab to access zkEVM, right next to Proof-os-Stake:
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:32]:
```suggestion
On the [Polygon Wallet Suite website](https://wallet-staging.polygon.technology/), you can find a tab to access zkEVM, right next to the existing Proof-of-Stake tab:
```

File: docs/zkEVM/wallet/usage-guide.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+<div align="center">
+<img src={useBaseUrl("img/zkevm/zkwallet-1.jpg")} width="1000" height="1000"/>
+</div>
+
+Clicking on **Bridge** will lead you to the zkEVM environment where you will be able to transfer assets from zkEVM to Mainnet/Testnet:
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:38]:
```suggestion
Click on the **Bridge** module to access the zkEVM environment where you will be able to transfer assets from zkEVM Testnet and Mainnet Beta:
```

File: docs/zkEVM/wallet/usage-guide.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+<div align="center">
+<img src={useBaseUrl("img/zkevm/bridge2.jpg")} width="800" height="800"/>
+</div>
+
+On the same page, you can also see the current pending transactions and the complete ones, on the right.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:44]:
```suggestion
On the right hand side of the page, you can view the recent transactions, including transactions that are pending.
```

File: docs/zkEVM/wallet/usage-guide.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+After you proceed with transaction on the bridge, Metamask will ask you to confirm it.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:46]: I'd add a simple step before this to demonstrate how to transact which explains what you can bridge.

File: docs/zkEVM/wallet/usage-guide.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+<div align="center">
+<img src={useBaseUrl("img/zkevm/metamask1.jpg")} width="400" height="400"/>
+</div>
+
+You might need to wait a few minutes and your transaction will be complete. The **Transactions** button on the left menu, will take you to a page where you will see all your past transactions as well as the pending ones.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:52]:
```suggestion
Please allow a few moments for your transaction to be processed. Once completed, you can view all of your past and pending transactions by clicking on the "Transactions" button located in the left menu.
```

File: docs/zkEVM/wallet/usage-guide.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+<div align="center">
+<img src={useBaseUrl("img/zkevm/transaction-history.jpg")} width="1000" height="1000"/>
+</div>
+
+:::tip Connecting to zkEVM
+
+You will prompted to add the **Polygon zkEVM** network to Metamask.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:60]:
```suggestion
You will be prompted to add the Polygon zkEVM network to Metamask.
```

File: docs/zkEVM/wallet/usage-guide.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+## zkEVM Support
+
+If you need help with anything related to the Polygon zkEVM, you can raise a ticket on [Polygon Support](https://support.polygon.technology/support/tickets/new) portal, and check out the [Knowledge base](https://support.polygon.technology/support/solutions/folders/82000694871) to view the most common queries about the zkEVM. Additionally, you can reach out to the support team available on the **#zkevm-support** channel on the [Polygon Discord server](https://discord.com/invite/XvpHAxZ). Instructions for raising a zkEVM testnet support ticket are as follows:
+
+1. Join the **Polygon Discord** server [here](https://discord.gg/0xPolygon).
+2. Accept the invite sent via DM.
+3. Take the **Member** role under **#roles**.
+4. Navigate to the **#zkevm-support** channel.
+
+You can now contact the zkEVM support staff with your questions and concerns. We will actively monitor for issues and work to resolve them as soon as possible.
```
Comment (DannyS03) [inline docs/zkEVM/wallet/usage-guide.md:85]:
```suggestion
Please feel free to contact zkEVM support with any questions or concerns you may have.
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
Thanks @cerberushades!

Added some suggestions.

File: docs/zkEVM/wallet/usage-guide.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
+---
+id: usage-guide
+title: Polygon ZkEVM Wallet Usage Guide
+sidebar_label: Usage Guide
```
Comment (ashucoder9) [inline docs/zkEVM/wallet/usage-guide.md:4]:
```suggestion
sidebar_label: Bridge to zkEVM
```

**Review by ashucoder9 — APPROVED** [review]
Thanks @cerberushades!

**Pull request merged**